src/typical.ts

```diff
@@ -76,32 +82,40 @@ async function perform(
   }
 }
 
-function* editor(edits: any) {
-  for (const edit of edits) {
+function* editor(edits: ReadonlyArray<string>) {
+  for (const snippet of edits) {
     yield {
-      op: (node: any) => requestAnimationFrame(() => (node.textContent = edit)),
+      op: (node: HTMLElementTagNameMap[Wrapper]) => requestAnimationFrame(() => (node.textContent = snippet)),
+
+      opCode: (node: HTMLElementTagNameMap[Wrapper]) => {
+        const nodeContent = node.textContent || '';
```

Comment:
`node.textContent` can potentilly be `null` and therefore would error when `.length` is accessed on the line below.